ansible/tasks/oauth2-proxy.yml

```diff
@@ -1,10 +1,24 @@
-- name: "download oauth2_proxy"
-  get_url:
-    url: "https://github.com/svsticky/oauth2_proxy/releases/download/v2.3/oauth2_proxy"
-    dest: "/usr/local/bin/oauth2_proxy"
-    checksum: "sha256:82a76b75c6f5d7986c0f6a9836818d1dcd00bcf0bc2a7e36dafcbaf9e980b5df"
+- name: "download and unarchive oauth2_proxy"
+  unarchive:
+    src: "https://github.com/pusher/oauth2_proxy/releases/download/v5.0.0/oauth2_proxy-v5.0.0.linux-amd64.go1.13.6.tar.gz"
+    remote_src: "yes"
```

Comment:
Maybe just checksum the tarball instead of the binary?

Reply:
+1